templates/swagger/v1_json.tmpl

```diff
@@ -21,7 +21,7 @@
     },
     "version": "1.1.1"
```

Comment:
Could you also fill the version with the corresponding var ?

Reply:
Yes, however I'd prefer to attempt to make an attempt in another PR.

Reply:
Yes, however I'd prefer to make an attempt in another PR.

Reply:
Yes, but I'd prefer to make an attempt in another PR.